docs/spec/governance/state.md

```diff
@@ -2,47 +2,58 @@
 
 ## State
 
-### Procedures
+### Procedures and base types
 
 `Procedures` define the rule according to which votes are run. There can only 
 be one active procedure at any given time. If governance wants to change a 
 procedure, either to modify a value or add/remove a parameter, a new procedure 
 has to be created and the previous one rendered inactive.
 
+
 ```go
+type Procedure struct {
+  VotingPeriod      int64               //  Length of the voting period. Initial value: 2 weeks
+  MinDeposit        sdk.Coins           //  Minimum deposit for a proposal to enter voting period. 
+  Threshold         rational.Rational   //  Minimum propotion of Yes votes for proposal to pass. Initial value: 0.5
+  Veto              rational.Rational   //  Minimum value of Veto votes to Total votes ratio for proposal to be vetoed. Initial value: 1/3
+  MaxDepositPeriod  int64               //  Maximum period for Atom holders to deposit on a proposal. Initial value: 2 months
+  GovernancePenalty sdk.Rat               //  Penalty if validator does not vote
```

Comment:
Maybe this needs to go in the slashing module?  Not sure, haven't really looked at that module.

Reply:
hmm I don't think so. This is slashing specific to the gov module. Gov module should not overlap in staking module. Or we are abstracting slashing away from staking?